integrations/adobe-analytics/lib/chromecast-heartbeat.js

```diff
@@ -0,0 +1,202 @@
+var trample = require('@segment/trample');
+
+var chromecastHeartbeat = {
+
+  /**
+  * Returns a mediaMetadata Object 
+  * @param {Track} track
+  * @returns {mediaMetadata} Maps the contextValue prop if it exists in the track.properties
+  */
+  extractMediaMetadata: function (track) {
+    debugger;
```

Comment:
```suggestion
```

Reply:
going to run more local test later today and then remove this!